infrastructures/infrastructure-azure/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AzureInfrastructure.java

```diff
@@ -520,15 +520,21 @@ public void notifyAcquiredNode(Node node) throws RMException {
 
     @Override
     public String getDescription() {
-        return "AzureInfrastructure handles nodes from Microsoft Azure.";
+        return "Handles nodes from Microsoft Azure.";
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
     public String toString() {
-        return getDescription();
+        return String.format("AzureInfrastructure numberOfInstances: [%s], numberOfNodesPerInstance: [%s], imageOSType: [%s], vmSizeType: [%s], region: [%s], maxBudget: [%s]",
```

Comment:
better use this.getClass().getSimpleName()

Reply:
Same remark for this project. Cannot use this.getClass().getSimpleName() as it ads unwanted characters